sys/include/psa_crypto/psa/crypto.h

```diff
@@ -2476,7 +2488,9 @@ psa_status_t psa_hash_update(psa_hash_operation_t *operation,
 psa_status_t psa_hash_verify(psa_hash_operation_t *operation,
                              const uint8_t *hash,
                              size_t hash_length);
+#endif /* IS_USED(MODULE_PSA_HASH) */
```

Comment:
Keeps the style consistent
```suggestion
#endif /* MODULE_PSA_HASH */
```